code/game/gamemodes/cult/runes.dm

```diff
@@ -621,7 +621,7 @@ structure_check() searches for nearby cultist structures required for the invoca
 		if(mob_to_revive.ghost_can_reenter())
 			mob_to_revive.grab_ghost()
 
-	if(!mob_to_revive.get_ghost() || mob_to_revive.client && mob_to_revive.client.is_afk())
+	if(!mob_to_revive.get_ghost() && !mob_to_revive.client || mob_to_revive.client.is_afk())
```

Comment:
please put brackets around this to make it easier to read

Reply:
applied